src/Panels/utils.ts

```diff
@@ -0,0 +1,8 @@
+export function getNonce() {
```

Comment:
why can't we use uuid instead?

Reply:
It's recommended way by vs code team.

Ref: https://github.com/microsoft/vscode-extension-samples/blob/dfb20f12d425bad2ede0f1faae25e0775ca750eb/webview-view-sample/src/extension.ts#L117-L123

Reply:
That is just an example, it does the job.. we can use uuid as it is already included within the dependency of nightwatch